apps/openmw/main.cpp

```diff
@@ -223,6 +228,10 @@ bool parseOptions (int argc, char** argv, OMW::Engine& engine, Files::Configurat
     if (!variables["skip-menu"].as<bool>() && variables["new-game"].as<bool>())
         Log(Debug::Warning) << "Warning: new-game used without skip-menu -> ignoring it";
 
+    const std::vector<std::string>& weaponTypes = variables["weapon-type"].as<Files::EscapeStringVector>().toStdStringVector();
```

Comment:
Nobody is really sure if overriding weapon types via configs is the right way to dehardcode them. Before any abrupt decisions are acknowledged in the codebase and it doesn't take too much effort *not* to have this, I'd reconsider having a stopgap solution this intrusive.